### PR TITLE
replace boilerplate 360 video example with new glitch

### PR DESCRIPTION
**Pull request by machenmusik**
Per discussion on aframevr/aframe#3210.

### CODE REVIEW

File: src/_data/examples.yml [hunk 1]
```diff
@@ -35,11 +35,9 @@ showcase:
 
 - section: showcase
   slug: videosphere
-  scene_url: https://aframe.io/aframe/examples/boilerplate/360-video/
-  source_url: https://github.com/aframevr/aframe/blob/v0.7.0/examples/boilerplate/360-video/index.html
+  scene_url: https://aframe-360-video-example.glitch.me/
+  source_url: https://glitch.com/~aframe-360-video-example
```
Comment (cvan) [inline src/_data/examples.yml]: https://glitch.com/edit/#!/aframe-360-video-example is the actual URL for View Source

Comment (cvan) [reply src/_data/examples.yml]: probably makes most sense to also link directly to the `index.html` file: https://glitch.com/edit/#!/aframe-360-video-example?path=index.html:1:0

**Comment by cvan** [comment]
https://glitch.com/edit/#!/aframe-360-video-example?path=multi-source-detail.html:33:0 <-- thanks for doing this investigative work. I think this is probably best suited as an HTML comment (and `console.warn` calls where appropriate).

File: src/_data/examples.yml (hunk 1 above)
Comment (machenmusik) [reply src/_data/examples.yml]: I was following the example of the other two glitches I saw in there for consistency ... and there is some useful information in the README that gets displayed.

**Comment by machenmusik** [comment]
I am not sure `console.warn` can be done effectively, as IIRC there are not clear indicators (e.g. in most failure cases mentioned video plays fine but texture is black and there is no indication)

**Comment by machenmusik** [comment]
(As HTML comment - done)

**Pull request merged**